content/applications/sales/sales/invoicing/milestone.rst

```diff
@@ -2,32 +2,237 @@
 Invoice project milestones
 ==========================
 
-Milestone invoicing can be used for expensive or large-scale projects,
-with each milestone representing a clear sequence of work that will
-incrementally build up to the completion of the contract. This invoicing
-method is comfortable both for the company which is ensured to get a
-steady cash flow throughout the project lifetime and for the client who
-can monitor the project's progress and pay in several installments.
+Invoicing based on project milestones can be used for expensive or large-scale projects. The series
+of milestones in a project represent a clear sequence of work that will inevitably result in the
+completion of a project and/or contract.
+
+This method of invoicing ensures the company gets a consistent flow of money throughout the lifetime
+of the project. Customers can closely monitor every phase of the project's development as it
+happens, in addition to paying a large bill in several installments, instead of all at once.
 
 Create milestone products
 =========================
 
-In Odoo, each milestone of your project is considered as a product. To
-configure products to work this way, go to any product form.
+In Odoo, each milestone of a project is considered as an individual product.
+
+To create and/or configure products to work like this, first navigate to :menuselection:`Sales app
```

Comment:
Because the instruction is explicitly navigational, the conjunction we want here is "and" because it includes the creation AND configuration of products to work as milestones.

```suggestion
To create and configure products to work like this, first navigate to :menuselection:`Sales app
```